content/release-10-16-0/datahub-10-16-0-bundle/10_16_0.md

```diff
@@ -0,0 +1,39 @@
+---
+weight: 40
+title: Release 10.16.0
+layout: bundle
+---
+
+Cumulocity IoT DataHub Release 10.16 includes the following improvements, limitations, and known issues:
+
+### Authentication against Azure data lake via Azure Active Directory
+
+When using Azure Storage as data lake, Azure Active Directory credentials can be used for authentication.
+
+### Configuration of offloading frequency
+
+Per default an offloading run is executed once an hour, for each hour of the day. The offloading frequency can be configured more fine-granular by selecting at which hours of the day an offloading run will be executed.
+
+### Minor UI enhancements
+
+The offloading page has been enhanced with status information. It provides the status of the most recent offloading and compaction runs for an offloading configuration. Regarding the definition of an offloading configuration, the additional filter predicate step provides examples for filter predicates.
+
+### Limitations
+
+|<div style="width:250px">Description</div>
+|:---
+|Mixed usage of uppercase and lowercase characters for an attribute name as well as series names in the documents is not supported.|
```

Comment:
For series, we have fixed this in 10.16.0.9: https://itrac.eur.ad.sag/browse/CDH-1810

We should put that under FEATURES and change the limitation to only mention attribute name (which is still an issue, waiting for the required Dremio fix)

Reply:
Let's check that together...

Reply:
@cheinz-sag @doernemt Can I merge this PR of do you still need to implement changes?

Reply:
Everything is clarified, PR is ready to be merged